src/translator_testing_model/schema/translator_testing_model.yaml

```diff
@@ -0,0 +1,479 @@
+    abstract: true
+    tree_root: true
+    slots:
+      - id
+      - name
+      - description
+
+  TestMetadata:
+    description: >-
+      Represents metadata related to (external SME, SMURF, Translator feedback, 
+      large scale batch, etc.) provenance of test assets, cases and/or suites.
+    is_a: TestEntity
+    slots:
+      - test_source
+      - test_reference
+      - test_objective
+
+  TestAsset:
+    description: >-
+      Represents a Test Asset, which is a test case agnostic
+      semantic specification of a Translator test target.
+    is_a: TestEntity
+    slots:
+      - input_id
+      - input_name
+      - output_id
+      - output_name
+      - expected_output
+      - test_issue
+      - semantic_severity
+      - in_v1
+      - well_known
+    slot_usage:
+      id:
+        aliases: ["AssetIdentifier"]
+        range: uriorcurie
+
+  QueryAnswerPair:
+    description: >-
+      Represents a QueryAnswerPair specification of a Test Asset
+#    aliases: [""]
+    is_a: TestAsset
+  
+  TestEdgeData:
+    description: >-
+      Represents a single Biolink Model compliant instance of an edge that can be used for testing.
+    is_a: TestAsset
+
+  TestCase:
+    description: >-
+      Represents a static specification of a Test Case
+    is_a: TestEntity
+    slots:
+      - mode
+      - inputs
+      - outputs
+      - preconditions
+
+  TestCaseSpecification:
+    description: >-
+      Parameterized declaration of a Test Case generator which dynamically
+      generates Test Cases from Test Assets using applicable heuristics.
+    is_a: TestEntity
+
+  TestSuite:
+    is_a: TestEntity
+    description: >-
+      Specification of a set of Test Cases, one of either with a
+      static list of 'test_cases' or a 'test_case_specification'
+      (note: at least one or the other needs to be present).
+    slots:
+      - mode
+      - test_metadata
+      - test_persona
+      - test_cases
+      - test_case_specification
+
+  AcceptanceTestCase:
+    is_a: TestCase
+    description: >-
+      Lifting schema from Shervin's runner JSON here as an example.  This schema is not yet complete.
+    slot_usage:
+      inputs:
+        range: SemanticSmokeTestInput
+        required: true
+      outputs:
+        range: SemanticSmokeTestOutput
+        required: true
+
+  AcceptanceTestSuite:
+    is_a: TestSuite
+    description:
+
+  BenchmarkTestSuite:
+    description:
+     is_a: TestSuite
+     
+  StandardsComplianceTestSuite:
+    description: >-
+      Test suite for testing Translator components against releases of standards like TRAPI and the Biolink Model.
+    is_a: TestSuite
+
+  OneHopTestSuite:
+    description: >-
+      Test case for testing the integrity of "One Hop" knowledge graph retrievals sensa legacy SRI_Testing harness.
+    is_a: TestSuite
+
+  Input:
+    description: >-
+      Represents an input to a TestCase
+    is_a: TestEntity
```

Comment:
Can we clarify the difference between `TestEntity` and `TestAsset`?

Reply:
I'll add a better description for **TestEntity** but it is really just a utilitarian global abstract parent 'identification' class for **_all_** the top level model classes - metadata, assets, cases, suites, ... - in that all of these latter classes are the 'entities' (concepts) of the testing model, the instances for which need to be distinguished (by identification) from one another.

In contrast, a **TestAsset** is a concrete expression ('instance') of a "test case agnostic" Translator knowledge graph query expectation, _sensa_ Jenn Hadlock activity. I've assumed that one **TestAsset** can be used to specify/generate more than one **TestCase** and that a collection of related **TestCase** instances form a **TestSuite**.